packages/frontend/src/hooks/useOrganizationGroups.ts

```diff
@@ -110,7 +110,7 @@ export const useInfiniteOrganizationGroups = (
 };
 
 const createGroupQuery = async (data: CreateGroup) =>
-    lightdashApi<Group>({
+    lightdashApi<GroupWithMembers>({
```

Comment:
I assume nothing else needs to change on the frontned to handle this type change. Perhaps the response was not even used

Reply:
Correct, I could find no usage in the frontend.